### PR TITLE
Token silent renew 

### PR DESCRIPTION
**Pull request by ChamseddineBhd**
Should be merged after merging token-silent-renw branch in powsybl-deployment
Signed-off-by: ChamseddineBhd <chamseddine.benhamed@ensi-uma.tn>

### CODE REVIEW

File: src/utils/authentication/AuthService.js [hunk 1]
```diff
@@ -94,4 +97,11 @@ function handleSigninCallback(dispatch, history, userManagerInstance) {
     });
 }
 
-export {userManagerPromise, login, logout, dispatchUser, handleSigninCallback, getPreLoginPath}
+function handleSilentRenewCallback(dispatch, userManagerInstance) {
+    userManagerInstance.signinSilentCallback().catch(function (e) {
+        dispatch(setLoggedUser(null));
```
Comment (jonenst) [inline src/utils/authentication/AuthService.js:102]: What does this do ?

Comment (ChamseddineBhd) [reply src/utils/authentication/AuthService.js:102]: If there's an issue with the silent renew we should force our user to login explicitly, to do that we empty the user in the store.

Comment (jonenst) [reply src/utils/authentication/AuthService.js:102]: but this is executed in the iframe no ?

Comment (ChamseddineBhd) [reply src/utils/authentication/AuthService.js]: Alright

**Pull request merged**